.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es2020": true,
+        "mocha": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "script",
+        "ecmaFeatures": {
+            "impliedStrict": true
+        }
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/no-explicit-any": "error",
```

Comment:
if I understand well, it is already reported as a warning. Given that he build is not failing anyway, is it really useful to set it to error? https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-explicit-any.md#rule-details

Reply:
I think a warning has less affect on a developers mind that an error. Even if it doesn't fail the build it is still displayed in the problems view.

Reply:
I'm actually ok with a warning on this one, but I can see why it might be nice to get an explicit error too. But I'm good either way so long as it gets reported.

Reply:
why do we want to have an higher effect for this rule specifically? Why not all rules that are at warning level to pass them at error level?

Reply:
> pointing that the jira number is missing from the commit messages

of course. the jira has been created way after the PR

Reply:
It was the recommendation in the liked tutorial. see "Level 3: any is evil" section of the article linked in the description of the PR. I don't see a reason to make everything an error and as far as I understood it in previous comments you are also more suggesting to do as little changes to the default rules as possible.

Reply:
have you checked that `an unknown type or a library without typings is used and you want to be able to specify any` ?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-explicit-any.md#when-not-to-use-it

Have you noticed in the pointed article that it is mentioned:
`But disabling all anys requires a little further knowledge of TypeScript than the basics.`?

Personally I don't feel having enough knowledge. Do you other references than just one article mentioning that forbidding `any` everywhere is the right way to go?

Reply:
How do you convert `scaffoldUtils.createFoldersFromJSON(any)` given that the argument is provided by `JSON.parse()` which is returning `any` (see in exetnsionFunction.scaffoldProjectFromJson())?

Reply:
You can use `unknown` as type if I remember correctly.